src/core/scene/a-scene.js

```diff
@@ -313,9 +367,24 @@ module.exports = registerElement('a-scene', {
         if (this.isPlaying) {
           this.tick(time, timeDelta);
         }
-        this.effect.render(this.object3D, camera);
+
+        window.performance.mark('render-iteration-started');
```

Comment:
can get rid of the whitespace lines until `render-iteration-finished` and move the comment above this entire block such that it's clear this whole block is about PP